server/utils/TestingUtils/MongoDbManager.js

```diff
@@ -1,56 +1,56 @@
 
 // Extend the default timeout so MongoDB binaries can download
 jest.setTimeout(60000);
 
-// List all of your collection names here - I'll add some examples
-const COLLECTIONS = ['completeCollection'];
-
 class DBManager {
   constructor() {
-    this.db = null;
     this.server = new MongoMemoryServer(
         {
             instance : {
                 dbName : 'rpaFrameworkCommands'
             }
         });
-    this.connection = null;
   }
 
   // Spin up a new in-memory mongo instance
   async start() {
     const url = await this.server.getUri();
     process.env.MONGODB_URI = url;
 
-    this.connection = await MongoClient.connect(url, {
-      useNewUrlParser: true,
-      useUnifiedTopology: true,
-    });
-
-    this.db = this.connection.db(await this.server.getDbName());
-    await this.db.createCollection('completeCollection');
-    let testDocuments = [
-        {"Application":"MS Excel","Task":"Open Application","Code":"Open Application","inputVars":{"visible":"Boolean","display_alerts":"Boolean"}},
-        {"Application":"Browser","Task":"Open Browser","Code":"Open Browser","inputVars":{"url":"String","browser":"String","alias":"String","remote_url":"Boolean","desired_capabilities":"String","ff_profile_dir":"String","options":"String","service_log_path":"String","executable_path":"String"}}
-    ];
-    await this.db.collection('completeCollection').insertMany(testDocuments);
+    const conn = await mongoose.createConnection(); // just create connection instance
+    const Tasks = conn.model('rpa-tasks'); // define model
+    conn.openUri(process.env.MONGODB_URI, { useNewUrlParser: true, useUnifiedTopology: true });
+    Tasks.createCollection()
+      .then(() => {
+        Tasks.insertMany(this.mockDocuments());
+      });
   }
 
   // Close the connection and halt the mongo instance
   stop() {
-    this.connection.close();
+    mongoose.disconnect();
     return this.server.stop();
   }
 
-  // Remove all documents from the entire database - useful between tests
-  cleanup() {
-    return Promise.all(COLLECTIONS.map((c) => this.db.collection(c).deleteMany({})));
-  }
-
-  log(input) {
-    console.log(input);
+  mockDocuments() {
+    const Task = mongoose.model('rpa-tasks');
+    const Variable = mongoose.model('rpa-tasks-variable');
+  
+    let var1 = new Variable({ Name: 'taste', Type: 'String', Required: true });
+    let var2 = new Variable({ Name: 'isHealthy', Type: 'Boolean', Required: false });
+    let task1 = new Task({ Application: 'My fancy Cookbook', Task: 'Taste Recipe', Code:'Taste Recipe', InputVars: [ var1, var2 ] });
+  
+    let var3 = new Variable({ Name: 'testVar', Type: 'String', Required: true });
+    let var4 = new Variable({ Name: 'anotherVar', Type: 'String', Required: true });
+    let var5 = new Variable({ Name: 'lastInput', Type: 'String', Required: true });
+    let var6 = new Variable({ Name: 'outputVar', Type: 'String', Required: true, InfoText: 'Output variable' });
```

Comment:
Maybe these vars can have better naming. I see, that using them as mock Objects, naming is not that important